docker/resources/ssl/README.md

```diff
@@ -0,0 +1,12 @@
+This folder contains SSL material for the docker composition default FQDN.
+
+This is how the certificate was created:
+```
+openssl req -new -newkey rsa:2048 -x509 -days 3650 -extensions v3_ca -keyout ca.pem -out ca.crt
+openssl rsa -in ca.pem -out ca.key
+openssl genrsa -out georchestra.mydomain.org.key 2048
+openssl req -key georchestra.mydomain.org.key -new -out georchestra.mydomain.org.csr
+openssl x509 -req -days 3650 -in georchestra.mydomain.org.csr -CA ca.crt -CAkey ca.key -CAcreateserial -out georchestra.mydomain.org.crt
```

Comment:
You don't need any certs with `georchestra-127-0-0-1.traefik.me`.

Reply:
you don't need to generate them, but you still need to grab them from traefik.me